packages/frontend/src/components/accounts/RecoverAccount.js

```diff
@@ -140,7 +140,7 @@ const RecoverAccount = (props) => {
                 <Option>
                     <Header icon={PhraseIcon}><Translate id='recoverAccount.phrase.title'/><br/><Translate id='recoverAccount.actionType'/></Header>
                     <P><Translate id='recoverAccount.phrase.desc'/></P>
-                    <Button to={`/recover-seed-phrase${props.history.location.search}`} onClick={()=> Mixpanel.track("IE Click seed phrase recovery button")}><Translate id='button.recoverAccount' /></Button>
+                    <Button to={`/recover-seed-phrase${props.history.location.search}`} data-test-id="recoverAccountWithPassphraseButton" onClick={()=> Mixpanel.track("IE Click seed phrase recovery button")}><Translate id='button.recoverAccount' /></Button>
```

Comment:
:nit: not a blocker, but please feel free to make props span multiple lines as we add new entries to them (less likely to cause conflicts that need manual resolution as we add/remove/change in the future).

Not a big deal, we'll be adopting a full set of ESLint rules to manage formatting Soon:tm: that will make this an implicit requirement via eslint.